rust/src/tests/rmdir.rs

```diff
@@ -100,6 +103,13 @@ fn has_mount_cap(_: &Config, _: &Path) -> anyhow::Result<()> {
     if !Uid::effective().is_root() && ctl.value()? == CtlValue::Int(0) {
         anyhow::bail!("process doesn't have the rights to mount the dummy file system")
     }
+    if !Uid::effective().is_root() &&
+        !OsStr::from_bytes(&Command::new("lsvfs").output().unwrap().stdout)
+            .to_string_lossy()
+            .contains("nullfs")
```

Comment:
Wouldn't it be better if we put this part in a variable?

Reply:
Which part?  I deliberately didn't put the command output in a variable like this:
```rust
let lsvfs = Command::new("lsvfs").output()...;
if !Uid::effective().is_root() && lsvfs.contains("nulllfs") {
```
because if I had done that, then the "lfvfs" command would always be executed.  Doing it this way ensures that "lsvfs" will only be executed if the process _isn't_ running as root.

Reply:
Oh OK I understand